### PR TITLE
Use skyra env utilities

### PR DESCRIPTION
**Pull request by KrishAgarwal2811**
This shifts the config system from using a `config.ts` file to `.env` files.

### CODE REVIEW

File: src/.env.development [hunk 1]
```diff
@@ -0,0 +1,11 @@
+EVLYN_HOST='localhost'
+EVLYN_PORT=3100
+
+CLIENT_OWNERS=''
+CLIENT_PREFIX='ad.'
+CLIENT_VERSION='v5.0.0'
+
+CLIENT_PRESENCE_NAME='ad. help'
```
Comment (kyranet) [inline src/.env.development:8]:
```suggestion
CLIENT_PRESENCE_NAME='ad.help'
```

File: src/.env [hunk 2]
```diff
@@ -0,0 +1,11 @@
+EVLYN_HOST=
+EVLYN_PORT=
+
+CLIENT_OWNERS=''
+CLIENT_PREFIX='a.'
+CLIENT_VERSION='v5.0.0'
+
+CLIENT_PRESENCE_NAME='a. help'
```
Comment (kyranet) [inline src/.env:8]:
```suggestion
CLIENT_PRESENCE_NAME='a.help'
```

File: src/.env [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+CLIENT_PRESENCE_TYPE='LISTENING'
+
+DISCORD_TOKEN=''
```
Comment (kyranet) [inline src/.env:11]: Missing EOF:
```suggestion
DISCORD_TOKEN=''

```

File: src/.env.development [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+CLIENT_PRESENCE_TYPE='LISTENING'
+
+DISCORD_TOKEN=''
```
Comment (kyranet) [inline src/.env.development:11]: Missing EOF:
```suggestion
DISCORD_TOKEN=''

```

File: README.md [hunk 5]
```diff
@@ -34,7 +34,7 @@ $ yarn pm2:start
 
 **Framework links**
 
--   [Klasa's Website](https://klasa.js.org)
+-   [Sapphire's Website](https://sapphirejs.com/)
```
Comment (kyranet) [inline README.md:37]: Revert

File: src/lib/AlestraClient.ts [hunk 6]
```diff
@@ -1,7 +1,6 @@
 import { WebsocketClient } from '#lib/websocket/WebsocketClient';
-import { ENABLE_EVLYN } from '#root/config';
 import { SapphireClient } from '@sapphire/framework';
 
 export class AlestraClient extends SapphireClient {
-	public websocket = ENABLE_EVLYN ? new WebsocketClient(this) : null;
+	public websocket = new WebsocketClient(this);
```
Comment (kyranet) [inline src/lib/AlestraClient.ts:5]: Either keep the check, or drop the line (and the class) entirely. Right now this change would force Evlyn to be up, even in development mode.

File: src/config.ts [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+import type { ActivityTypes } from 'discord.js/typings/enums';
+
+export const ROOT_FOLDER = join(getRootData().root, '..');
+setup(join(ROOT_FOLDER, 'src', '.env'));
+
+export const DEV = envParseString('NODE_ENV') !== 'production';
+export const EVLYN_HOST = envParseString('EVLYN_HOST');
+export const EVLYN_PORT = 3100;
+
+export const OWNERS: string[] = envParseArray('CLIENT_OWNERS');
```
Comment (kyranet) [inline src/config.ts:14]:
```suggestion
export const OWNERS = envParseArray('CLIENT_OWNERS');
```

File: src/.env.development [hunk 8]
```diff
@@ -0,0 +1,11 @@
+EVLYN_HOST='localhost'
+EVLYN_PORT=3100
+
+CLIENT_OWNERS=''
+CLIENT_PREFIX='ad.'
+CLIENT_VERSION='v5.0.0'
```
Comment (kyranet) [inline src/.env.development:6]:
```suggestion
CLIENT_VERSION='v5.0.0-dev'
```

File: src/lib/AlestraClient.ts (hunk 6 above)
Comment (KrishAgarwal2811) [reply src/lib/AlestraClient.ts]: removed the websocket client creation and the following code will be removed in a follow up PR

Comment (KrishAgarwal2811) [reply src/lib/AlestraClient.ts]: added the changes in the same PR

**Review by kyranet — APPROVED** [review]

**Review by favna — APPROVED** [review]

**Pull request merged**